andela/pippin-ah-backend
# CH(Coveralls): integrate coveralls with Authors Haven

#### What does this PR do?
- Integrate coveralls with Authors Haven
#### Description of Task to be completed?
- [x] install nyc and mocha-lcov-reporter
- [x] modified test script to inlude nyc reporting
- [x] install coveralls package
- [x] add after_success script in .travis.yml file to run coverage script
- [x] add coverage script in package.json file
- [x] add nyc_output in .gitignore file
- [x] add coverage in .gitignore file
#### How should this be manually tested?
- click on [coveralls](https://github.com/andela/pippin-ah-backend/tree/CH/162727548/coveralls-integration) to see the badge 
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[Finishes #162727548]
